utilities/src/main/java/io/syndesis/qe/utils/RestUtils.java

```diff
@@ -103,7 +108,8 @@ private static HttpClient createAllTrustingClient() throws RestClientException {
             final SSLContextBuilder builder = new SSLContextBuilder();
             builder.loadTrustMaterial((TrustStrategy) (X509Certificate[] chain, String authType) -> true);
             final SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(
-                    builder.build());
+                    builder.build(),
+                    new NoopHostnameVerifier()); // needed to connections to API Provider integrations
```

Comment:
@tplevko not sure this is ok, should I rewrite it to have a specific client for api provider?

Reply:
@asmigala I believe this won't be an issue, as this client is solely used for testing purposes